### ISSUE TITLE
As a developer I want to have a graphical inspector,

### ISSUE THREAD

**Issue by Ferada** [Ferada/cl-cffi-gtk#7]
so I don't have to have SLIIME open.

Looking at the commercial Lisps here. And compare with the GTK-own inspector, which can see their own properties and values. Maybe it can be hooked into even?

**Comment by stacksmith** [comment]
@Ferada - have you looked at https://github.com/stacksmith/subtext 
Especially the REPL?  It is a mockup/demo, but the idea of text fragments that are connected to lisp code, can bind keys, etc... is really powerful for building things like inspectors...
I am doing a major rewrite of subtext, so don't spend too much of your time using the demo...

**Comment by Ferada** [comment]
Yes, I like it a lot. For me would definitely make sense to use the textual representation and have clickable links to expand to drill down, similar to how the SLIME inspector looks like.

Well and all with keyboard navigation available too of course.

There should also somehow be an easy mechanism for users to override the display mechanism for custom classes like e.g. to display image etc.
